test/ruff.toml

```diff
@@ -0,0 +1,6 @@
+extend = "../pyproject.toml"
+ignore = [
+    "S101", # Assertions are OK in test code.
+    "N802", # Project names are used as test names thus causing conflicts with this rule.
+    "D",    # Only limited docstrings are required.
```

Comment:
In this project we will have to include more docstrings to keep information about what tests do there. For now remove "D" from this file ignore section, but leave file and ignore section itsel